package test16_exception_thread.lambda;

public class Array {
}
